modal/gpu.py

```diff
@@ -197,7 +204,9 @@ def parse_gpu_config(value: GPU_T) -> api_pb2.GPUConfig:
             gpu_type=gpu_type,
             count=count,
         )
-    elif value is None or value is False:
+    elif value is None:
         return api_pb2.GPUConfig()
     else:
-        raise InvalidError(f"Invalid GPU config: {value}. Value must be a string, a `GPUConfig` object, or `None`.")
+        raise InvalidError(
+            f"Invalid GPU config: {value}. Value must be a string or `None` (or a deprecated `_GPUConfig` object)."
```

Comment:
Why are we pointing users towards the private version of the object?

Reply:
`_GPUConfig` is just the base class of all the public objects

Reply:
Yeah just seems confusing to mention it in a public error message. Maybe "or a `modal.gpu` object, although these are deprecated"?

Reply:
sure sgtm, i can update